tests/rptest/wasm/cli_kafka_producer.py

```diff
@@ -1,32 +0,0 @@
-# Copyright 2021 Vectorized, Inc.
```

Comment:
Also I think this removing should be in another commit

Reply:
I think its ok since the commit removes the last reference to `KafkaCliProducer`